docs/guides/deploy-evm-svm-warp-route.mdx

```diff
@@ -57,10 +57,11 @@ Ensure you have the following installed and configured:
 
 7. Deploy the Ethereum Warp Route contract, which will also enroll the SVM router based on the yaml token config.
 
-   1. Check out the monorepo branch from this [PR#4447](https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4447)
-   2. Build the TS code from project root: `yarn && yarn build`
-   3. Run the CLI from source: `yarn workspace @hyperlane-xyz/cli hyperlane`
-   4. Create a YAML Warp Route configuration file using this template and steps:
+   1. Ensure you have latest version of the [Hyperlane CLI](/docs/reference/cli.mdx).
+   2. Create a Warp Route deployment config file with the CLI's config command: `hyperlane warp init `
+
+      The command provides a walkthrough, prompting you for configuration choices directly in the terminal.
```

Comment:
can we also add that they need to manually set the `foreignDeployment: <SVM address>` for the SVM chain? like the example down below